docs/style.json

```diff
@@ -20,7 +20,7 @@
     },
     "mapterhorn-hillshade": {
       "type": "raster-dem",
-      "url": "https://tunnel.optgeo.org/martin/mapterhorn",
+      "url": "pmtiles://https://download.mapterhorn.com/",
```

Comment:
this should be "url": "pmtiles://https://download.mapterhorn.com/planet.pmtiles",